sdk/dotnet/sdk/Dagger.SDK.SourceGenerator/Extensions/EnumerableExtensions.cs

```diff
@@ -3,23 +3,23 @@
 
 namespace Dagger.SDK.SourceGenerator.Extensions;
 
-public static class EnumerableExtensions
-{
-    public static IEnumerable<TSource> ExceptBy<TSource, TKey>(
-        this IEnumerable<TSource> source,
-        IEnumerable<TKey> second,
-        Func<TSource, TKey> keySelector
-    )
-    {
-        HashSet<TKey> keys = [.. second];
-
-        foreach (TSource element in source)
-        {
-            if (keys.Contains(keySelector(element)))
-            {
-                continue;
-            }
-            yield return element;
-        }
-    }
-}
+// public static class EnumerableExtensions
+// {
+//     public static IEnumerable<TSource> ExceptBy<TSource, TKey>(
+//         this IEnumerable<TSource> source,
+//         IEnumerable<TKey> second,
+//         Func<TSource, TKey> keySelector
+//     )
+//     {
+//         HashSet<TKey> keys = [.. second];
+//
+//         foreach (TSource element in source)
+//         {
+//             if (keys.Contains(keySelector(element)))
+//             {
+//                 continue;
+//             }
+//             yield return element;
+//         }
+//     }
+// }
```

Comment:
We don't need it after we upgraded to .NET8.0 right?

Reply:
This is what I am having trouble in understanding 100%. .NET8 introduced this, but we made because of netstandard, but netstandard is considered legacy while the docs still recommend netstandard for source generators. I am stuck in the middle of knowing what we should really do to push forward. Can we really forget about netstandard. (I hope we can)

Reply:
https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1#when-to-target-netx0-vs-netstandard

There is so much explanation discussed here, I really don't like netstandard lol

Reply:
I did some more reading, supposedly, Visual Studio may have issues, because Visual Studio is using .NET Framework. This means Source Generator will only work on IDE of VS Code & JetBrains Rider. 

This is actually quite annoying... I am not sure if they have plans for Visual Studio to use the dotnet build pipeline for source generator like vscode.

Reply:
I think multi target could be an option. `<TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>` and 

`#if NETSTANDARD2_0` for the `ExceptBy`

Reply:
I have put this back, there are some issues and it basically boils down to different IDEs triggering source generation. A bit annoying.